Sources/SwiftSyntax/SwiftSyntaxCompatibility.swift

```diff
@@ -1,903 +0,0 @@
-//===----------------------------------------------------------------------===//
-//
-// This source file is part of the Swift.org open source project
-//
-// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
-// Licensed under Apache License v2.0 with Runtime Library Exception
-//
-// See https://swift.org/LICENSE.txt for license information
-// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
-//
-//===----------------------------------------------------------------------===//
-
-// This file provides compatiblity aliases to keep dependents of SwiftSyntax building.
-// All users of the declarations in this file should transition away from them ASAP.
-
-@available(*, deprecated, renamed: "ImportPathSyntax")
-public typealias AccessPathSyntax = ImportPathSyntax
-
-@available(*, deprecated, renamed: "ImportPathComponentSyntax")
-public typealias AccessPathComponentSyntax = ImportPathComponentSyntax
-
-@available(*, deprecated, renamed: "WithAttributesSyntax")
-public typealias AttributedSyntax = WithAttributesSyntax
-
-public extension ArrayTypeSyntax {
```

Comment:
I don’t see a generated compatibility function for `ArrayTypeSyntax`. Did you mix it up with `ArrayExprSyntax`?

Reply:
Nope, I think I just missed to add the deprecatedName for `ArrayTypeSyntax`

Reply:
https://github.com/apple/swift-syntax/pull/1751/files

Yeah we missed it. 
Another good reason for generating things 🔥